README.md

```diff
@@ -1,216 +1,265 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) Mobile Application
+# Compose Multiplatform mobile application
 
-Use this template to start developing your own Compose Multiplatform application targeting Android and iOS (Alpha).
+> Compose Multiplatform is in Alpha. It may change incompatibly and require manual migration in the future.
```

Comment:
Compose MultiPlatform for iOS is in Alpha. We have to add iOS always as Desktop is considered stable

Reply:
@danil-pavlov I think this still needs to be addressed?

Reply:
@danil-pavlov Could you please update to `Compose Multiplatform for iOS is in Alpha`?

Reply:
```suggestion
> Compose Multiplatform for iOS is in Alpha. It may change incompatibly and require manual migration in the future.
```